### ISSUE TITLE
Missing ProjectDescriptionHelpers when running tuist init

### ISSUE THREAD

**Issue by fortmarek** [tuist/tuist#6773]
### What happened?

`tuist init` is broken. The regression was introduced in `4.28.0` as `tuist init` on `4.27.0` works just fine. It's likely

### How do we reproduce it?

1. Run `tuist init`

### Error log

```
We received an error that we couldn't handle:
    - Localized description: The operation couldn’t be completed. (_NIOFileSystem.FileSystemError error 1.)
    - Error: Unknown: Unable to get information about '/Users/marekfort/.local/share/mise/installs/tuist/4.28.0/bin/Tuist/ProjectDescriptionHelpers'. ('stat' system call failed with '(20) Not a directory'.)

If you think it's a legit issue, please file an issue including the reproducible steps: https://github.com/tuist/tuist/issues/new/choose
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
```

### Tuist version

4.28.0

**Comment by Jaesung-Jung** [comment]
same 4.28.1

**Comment by Playrom** [comment]
Using tuist 4.28.2 , which should have included the PR that solves this issue, the problem i still present

```
We received an error that we couldn't handle:
    - Localized description: The operation couldn’t be completed. (_NIOFileSystem.FileSystemError error 1.)
    - Error: Unknown: Unable to get information about '/Users/playrom/.local/share/mise/installs/tuist/4/bin/Tuist/ProjectDescriptionHelpers'. ('stat' system call failed with '(20) Not a directory'.)
```

**Comment by fortmarek** [comment]
Thanks @Playrom, you are right. I'm investigating this.

**Comment by fortmarek** [comment]
Should be fixed in `4.29.0`. I tried it with the latest version and `tuist init` works. We can reopen again if it doesn't for you.
